Monika After Story/game/event-handler.rpy

```diff
@@ -1499,11 +1499,10 @@ init python:
                 (Default: False)
         """
         if ev:
-
             if unlock:
                 ev.unlocked = True
 
-            if _random:
+            if _random and store.mas_bookmarks_derand.shouldRandom(ev.eventlabel):
```

Comment:
Why are bookmarks derand rules being enforced here?

The event APIs here are for helping us do common actions regardless of any topic restrictions. 
It's the caller's responsibility to make sure they aren't randoming a topic that shouldn't be randomed. Where are the problem lines that call this without checking? If it's truly a significant amount, we should just have a separate API for randoming a topic that also checks whatever restrictions we may have.